Dockerfile

```diff
@@ -12,7 +12,7 @@ ARG BAZEL_DOWNLOAD_URL=https://github.com/bazelbuild/bazel/releases/download
 ENV HOME=/root
 ENV PROJECT_DIR="${HOME}/PyDP"
 ENV PATH="/root/bin:${PATH}"
-ENV DP_SHA="1b1dc6639173c0a13613189ec21851604a4c7335"
+ENV DP_SHA="613c69ec6a7058c821b52ae6ad0ae04ad28270f1"
```

Comment:
SHA change needs to be added in ./build_pydp.sh and across all files in github actions.

Reply:
Done!